spec/features/user_views_author_page_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'User visits author page' do
+  it 'see author information and book list' do
+    author = create(
+      :author,
+      first_name: 'Foo',
+      last_name: 'Bar',
+      biography: 'some weird stuff'
+    )
+    create(
+      :book,
+      author: author,
+      title: 'very good title',
+      description: 'you shuld read it',
+      cover_url: 'http://example.com/cool_cover.jpg'
```

Comment:
You don't need to pass details of book explicitly, it is irrelevant for this test. You can rely on factory defaults.